### DISCUSSION TITLE
YHYPE profile should publicly sharable [POLL]

### DISCUSSION THREAD

**Discussion by athrvvvv** [yhype/yhype-feedback#10] [Polls]
### [YHYPE](https://yhype.me/) is really very <strong>Innovative</strong> and <strong>Futuristic</strong> concept. We should add useful feautures to it, before anyone else makes best out of it...
---
### I think YHype profile should be sharable, we should be able share our Github profiles's STACTICTICS as well as we should be able to share the statictics of any particular REPOSITORY.
---
### Example :- [Codetrace.com](https://codetrace.com/users/athrvvvv) here, you can see / visit my profile as well as see activities of my profile.
![Screenshot 2022-04-15 130425](https://user-images.githubusercontent.com/98272709/163540129-ac5604ad-7a26-4f28-9edf-cf3fe6242ed4.png)
---
[YHYPE](https://yhype.me/) should be upgraded with all the <strong> USEFUL</strong> features, before anyone else make it true. Come on, make YHYPE the new social media for devs.
---
<h3>whether you feel the feature will be amazing ot not!
POLL your answer now!

**Comment by ytakemon** [comment]
I want to second this and also share that it would be very useful for the scientific research community. It would be great if we can share stats to repos/tools with editors during manuscript review processes. This will greatly help us demonstrate to the journal editors whether a tool is already generated interest a the pre-publication stage and empirical evidence as to why journals should publish certain tools.
